.mailmap

```diff
@@ -0,0 +1,67 @@
+Felix Arntz <flixos90@git.wordpress.org> <flixos90@602fd350-edb4-49c9-b593-d223f7449a82>
+Gary Pendergast <pento@git.wordpress.org> <pento@602fd350-edb4-49c9-b593-d223f7449a82>
+Grzegorz Ziółkowski <gziolo@git.wordpress.org>
+Grzegorz Ziółkowski <gziolo@git.wordpress.org> <gziolo@602fd350-edb4-49c9-b593-d223f7449a82>
+Helen Hou-Sandi <helen@git.wordpress.org> <helen@602fd350-edb4-49c9-b593-d223f7449a82>
+Ian Dunn <iandunn@git.wordpress.org> <iandunn@602fd350-edb4-49c9-b593-d223f7449a82>
+Ian Stewart <iandstewart@git.wordpress.org> <iandstewart@602fd350-edb4-49c9-b593-d223f7449a82>
+Jake Spurlock <whyisjake@git.wordpress.org> <whyisjake@602fd350-edb4-49c9-b593-d223f7449a82>
+James Nylen <jnylen0@git.wordpress.org> <jnylen0@602fd350-edb4-49c9-b593-d223f7449a82>
+Jb Audras <audrasjb@git.wordpress.org> <audrasjb@602fd350-edb4-49c9-b593-d223f7449a82>
+Jeremy Felt <jeremyfelt@git.wordpress.org> <jeremyfelt@602fd350-edb4-49c9-b593-d223f7449a82>
+Joe Dolson <joedolson@git.wordpress.org> <joedolson@602fd350-edb4-49c9-b593-d223f7449a82>
+Joe Hoyle <joehoyle@git.wordpress.org> <joehoyle@602fd350-edb4-49c9-b593-d223f7449a82>
+Joe McGill <joemcgill@git.wordpress.org> <joemcgill@602fd350-edb4-49c9-b593-d223f7449a82>
+John Blackbourn <johnbillion@git.wordpress.org> <johnbillion@602fd350-edb4-49c9-b593-d223f7449a82>
+Jonathan Desrosiers <desrosj@git.wordpress.org> <desrosj@602fd350-edb4-49c9-b593-d223f7449a82>
+Jonny Harris <spacedmonkey@git.wordpress.org> <spacedmonkey@602fd350-edb4-49c9-b593-d223f7449a82>
+Jorge Costa <jorgefilipecosta@git.wordpress.org> <jorgefilipecosta@602fd350-edb4-49c9-b593-d223f7449a82>
+Juliette Reinders Folmer <jrf@git.wordpress.org>
+K. Adam White <kadamwhite@git.wordpress.org> <kadamwhite@602fd350-edb4-49c9-b593-d223f7449a82>
+Kelly Choyce-Dwan <ryelle@git.wordpress.org>
+Konstantin Obenland <obenland@git.wordpress.org>
+Konstantin Obenland <obenland@git.wordpress.org> <obenland@602fd350-edb4-49c9-b593-d223f7449a82>
+Lance Willett <lancewillett@git.wordpress.org> <lancewillett@602fd350-edb4-49c9-b593-d223f7449a82>
+Matias Ventura <matveb@git.wordpress.org> <matveb@602fd350-edb4-49c9-b593-d223f7449a82>
+Michael Adams <mdawaffe@git.wordpress.org>
+Pascal Birchler <swissspidy@git.wordpress.org> <swissspidy@602fd350-edb4-49c9-b593-d223f7449a82>
+Peter Wilson <peterwilsoncc@git.wordpress.org> <peterwilsoncc@602fd350-edb4-49c9-b593-d223f7449a82>
+Rachel Baker <rachelbaker@git.wordpress.org> <rachelbaker@602fd350-edb4-49c9-b593-d223f7449a82>
+Riad Benguella <youknowriad@git.wordpress.org> <youknowriad@602fd350-edb4-49c9-b593-d223f7449a82>
+Ryan McCue <rmccue@git.wordpress.org> <rmccue@602fd350-edb4-49c9-b593-d223f7449a82>
+Scott Taylor <wonderboymusic@git.wordpress.org> <wonderboymusic@602fd350-edb4-49c9-b593-d223f7449a82>
+Sergey Biryukov <sergeybiryukov@git.wordpress.org> <SergeyBiryukov@602fd350-edb4-49c9-b593-d223f7449a82>
+Tammie Lister <karmatosed@git.wordpress.org>
```

Comment:
Is this a mistake? It has no email to replace

Reply:
I notice there's several. Or is this just adding the name to cases where it's just the email?

Reply:
Yeah, thanks @ellatrix - in this case this line is correcting a name. An entry with the correct email listed `Tammy` as the name, and I eagerly matched it to `Tammie Lister` so that it would join Tammie's other contributions.